dev-requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest
 pytest-cov
 segtok>=1.5.7
 tensorflow==1.13.1
+pydocstyle
```

Comment:
You might want to pin this to at least a specific major version (same for other linters), otherwise tests might suddenly start failing for no obvious reason when these packages get updated.

Reply:
@obi1kenobi Noted; will pin the rest of the dev-requirements!